internal/loader/stdlib_dynamic_test.go

```diff
@@ -0,0 +1,39 @@
+package loader
+
+import (
+	"os"
+	"sync"
+	"testing"
+
+	"golang.org/x/tools/go/packages"
+)
+
+func TestStdlibDetectionNoBazel(t *testing.T) {
+	os.Unsetenv("GOPACKAGESDRIVER")
+
+	testCases := []struct {
+		pkgPath  string
+		isStdlib bool
+	}{
+		{"fmt", true},
+		{"net/http", true},
+		{"encoding/json", true},
+		{"vendor/golang.org/x/crypto/chacha20", true},
+		{"github.com/sourcegraph/scip-go", false},
+		{"golang.org/x/tools", false},
+		{"vendorapi.com/foo", false},
+	}
+
+	t.Run("DynamicLoading", func(t *testing.T) {
+		stdlibOnce = sync.Once{}
+		stdlibMap = nil
```

Comment:
Why is this initialization needed? Does the default initialization not cover this?